test/pipe.js

```diff
@@ -50,18 +50,25 @@ assert.equal(shell.error(), null);
 assert.equal(result.toString(), shell.cat('resources/uniq/pipeSorted').toString());
 
 // Synchronous exec
+var findCommand;
 if (process.platform === 'win32') {
+  // On Windows, we use the FIND command. FIND is semantically similar
+  // to POSIX’s grep(1) except it only matches string literals. Also,
+  // FIND requires the first argument, the search string, to be
+  // wrapped in double quotes when accessed via GetCommandLine().
+  // https://technet.microsoft.com/en-us/library/bb490906.aspx
+
+  // Get the full path to FIND. If we just exec('find'), Windows will
+  // try to run ./find.js with Windows Script Host.
+  findCommand = '"' + shell.which('FIND') + '"';
```

Comment:
Can you move this section out of the conditional? It'll make it easier when we merge this into #512.